### PR TITLE
Vet Clinic: Update Animal Table

### PR DESCRIPTION
**Pull request by julzedz**
In this project, I did the following:
- Added a new species column.
- Inserted new animal data.
- Updated the animal table within a transaction.
- Compute calculations using aggregate functions and GROUP BY.

Screen Shot 2022-12-17 at 9 37 06 AM

Screen Shot 2022-12-17 at 9 21 26 AM

Screen Shot 2022-12-17 at 9 21 26 AM

Screen Shot 2022-12-17 at 10 33 14 AM

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -2,9 +2,11 @@
...
-    id INT,
+    id int NOT NULL IDENTITY(1,1),
     date_of_birth DATE,
     escape_attempts INT,
     neutered BOOLEAN,
-    weight_kg DECIMAL
+    weight_kg DECIMAL,
+    species VARCHAR(100),
+    PRIMARY KEY (id)
 );
```
Comment (Munsa1) [inline schema.sql]: - [ ] In the schema file there should be the following statement as it is a requirement `ALTER TABLE animals ADD species varchar(255);`

File: queries.sql [hunk 2]
```diff
@@ -6,4 +6,15 @@ SELECT date_of_birth FROM animals WHERE name = 'Pikachu' OR name = 'Agumon';
 SELECT name, escape_attempts  FROM animals WHERE weight_kg > 10.5;
```
Comment (Munsa1) [inline queries.sql:6]: - [ ] In the queries file you need to have the following transactions, kindly do so

image
2
image

3
image

4
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @julzedz, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**